src/sync/chase_lev.rs

```diff
@@ -34,14 +34,11 @@
 //! stealer2.steal();
 //! ```
 
-// FIXME: all atomic operations in this module use a SeqCst ordering. That is
-//      probably overkill
```

Comment:
Could you replace this comment with a reference to the paper from which all atomic orderings are referencing? It's good to have something to double check against!